web-app/src/app/routes/alert/alert-notice/alert-notice.component.ts

```diff
@@ -691,6 +707,17 @@ export class AlertNoticeComponent implements OnInit {
   }
 
   onManageRuleModalOk() {
+    // @ts-ignore
+    if (this.ruleForm.invalid) {
+      // @ts-ignore
+      Object.values(this.ruleForm.controls).forEach(control => {
```

Comment:
hi, no suggest use the @ts-ignore to ignore warning, how about this `this.ruleForm?.invalid` or others

Reply:
Yes, you are right, I will optimize it, thanks for the reminder